thirdparty/CMakeLists.txt

```diff
@@ -0,0 +1,173 @@
+cmake_minimum_required(VERSION 3.5.1)
```

Comment:
I'd rather be moving further away from CMake tbh. Didn't someone have a Meson branch? ;-) But with the stuff from Debian cross forcing it into some semblance of sanity it's fine by me. ^_^